x-pack/plugins/infra/public/components/logging/log_text_stream/scrollable_log_text_stream_view.tsx

```diff
@@ -155,39 +165,54 @@ export class ScrollableLogTextStreamView extends React.PureComponent<
     return (
       <ScrollableLogTextStreamViewWrapper>
         {isReloading && (!isStreaming || !hasItems) ? (
-          <InfraLoadingPanel
-            width="100%"
-            height="100%"
-            text={
-              <FormattedMessage
-                id="xpack.infra.logs.scrollableLogTextStreamView.loadingEntriesLabel"
-                defaultMessage="Loading entries"
+          <>
+            {renderLoadingState ?? (
```

Comment:
With `renderLoadingState` being a render prop now, don't we need to call it (like we already do with `renderEmptyState`)?